RazorHtmlEmails.RazorClassLib/Views/RegistrationConfirmation.cshtml.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorHtmlEmails.RazorClassLib.Views
+{
+    public class RegistrationConfirmation : PageModel
+    {
+        private string UserName;
+
+        public RegistrationConfirmation(string userName)
+        {
+            UserName = userName;
+        }
+
+        public RegistrationConfirmation()
```

Comment:
Do we need this empty constructor? This email template requires userName so what we gonna show if an userName won't be passed?